app.js

```diff
@@ -6,11 +6,25 @@ var http = require('http').Server(app);
 var io = require('socket.io')(http);
 
 var questionsList = question.correctQuestions();
```

Comment:
why it's called `correctQuestions`? do you have `incorrectQuestions` somewhere too?

Strange name